python/requirements.txt

```diff
@@ -6,7 +6,6 @@ blinker==1.4
 certifi==2019.11.28
 chardet==3.0.4
```

Comment:
By using that command 
 python -m pip freeze  > requirements.txt

Reply:
But I think that determines the `requirements.txt` according to what is installed within your environment.
For example, I could quickly find random three items in this list that we don't actually have a dependency on: Twisted, zope.interface, PyGObject

Maybe you can run `python -m pip freeze` in a fresh environment.

I think this is an important item before to merging this pr.
This is because,  in case somebody (other than you) needs to execute those scripts, they would need wait for all the things in this list to be installed, which would slow down the process a lot.

Reply:
Fixed using pip-tools

Reply:
@gurkanindibay can you share a brief explanation on how we use `pip-tools` ? I still see the requirements.txt in the branch and I am quite confused.

Reply:
We still use requirements.txt . pip-tools has compile option I used pip-compile --generate-hashes option 
The steps I followwed to create requirements.txt is below
1. Create requirements.in file with required dependencies
2. create a virtual environment using pip to provide clean run
  python3 -m venv env
3. activate virtual environment
 source env/bin/activate
4. Go to directory which includes requirements.in
5. Run pip-compile with hash option
 pip-compile --generate-hashes
References:
https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/
https://github.com/jazzband/pip-tools

Reply:
It would be great if you document this list of items in a readme file

Reply:
https://github.com/citusdata/tools/issues/104